packages/cli/package.json

```diff
@@ -108,6 +108,13 @@
     "yn": "^4.0.0"
   },
   "devDependencies": {
+    "@backstage/backend-common": "^0.1.1-alpha.25",
+    "@backstage/cli": "^0.1.1-alpha.25",
```

Comment:
the cli depends on itself?

Reply:
![image](https://user-images.githubusercontent.com/4984472/97374830-91427480-18b9-11eb-8472-1dbc940c9fd1.png)

Reply:
![image](https://user-images.githubusercontent.com/3645856/97375204-7c1a1580-18ba-11eb-8edc-134f95ad76a7.png) 😂 `derpencency`